packages/contracts/contracts/test/CallbackHandlerHelperMock.sol

```diff
@@ -2,17 +2,23 @@
 
 pragma solidity 0.8.10;
 
-contract CallbackHandlerMockHelper {
+import {CallbackHandler} from "../core/component/CallbackHandler.sol";
+
+contract CallbackHandlerMockHelper is CallbackHandler {
     address callbackHandlerMockAddr;
 
-    constructor(address _callbackHandlerMockAddr) {
-        callbackHandlerMockAddr = _callbackHandlerMockAddr;
+    /// @notice Executes _handleCallback on the parens for testing purposes.
+    /// @param selector Any kind of selector in order to call fallback.
+    /// @param data Some arbitrary data. Mainly, you should pass msg.data to _handleCallback most of the time.
+    function handleCallback(bytes4 selector, bytes memory data) external returns (bytes4) {
+        bytes4 magicNumber = _handleCallback(selector, data);      
+        return magicNumber;
```

Comment:
Does the stack-saving practice apply here?

```suggestion
    function handleCallback(bytes4 selector, bytes memory data) external returns (bytes4 magicNumber) {
        magicNumber = _handleCallback(selector, data);      
```

Reply:
It's a test file, so I preferred to be more explicit.